Formula/r/rqlite.rb

```diff
@@ -1,8 +1,9 @@
 class Rqlite < Formula
   desc "Lightweight, distributed relational database built on SQLite"
   homepage "https://www.rqlite.io/"
-  url "https://github.com/rqlite/rqlite/archive/refs/tags/v8.36.3.tar.gz"
-  sha256 "c57e1499e013d69f2c0201a368546691537a67388e967c38c03948a9ea208b57"
+  url "https://github.com/rqlite/rqlite.git",
+    tag:      "v8.36.3",
+    revision: "92bcff8e0b50570294c3d13f6b598821ad3daff6"
```

Comment:
would be better to use tarball

Reply:
Tarball cannot be used if we want the commit hash.

Reply:
As mentioned, commit hash is a nice-to-have. I don't consider it critical to the release information.